templates/scc.yaml.in

```diff
@@ -16,3 +16,5 @@ seLinuxContext:
   type: RunAsAny
 users:
   - system:serviceaccount:{{ .Namespace }}:macvtap-cni
+volumes:
+  - "*"
```

Comment:
I don't know much about SCC, but could you me specific about the volumes you actually need? This seems like something that is unnecessarily lazy and can be difficult to fix in the future

Reply:
I don't think I can; this represents the type of volume, as per the [docs](https://docs.openshift.com/container-platform/4.10/rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.html): 
> Volumes is a white list of allowed volume plugins. FSType corresponds directly with the field names of a VolumeSource (azureFile, configMap, emptyDir). To allow all volumes you may use "*". To allow no volumes, set to ["none"].

I don't think I can tie this functionality to the type of volume - i.e. "this will only allow for `configMap` volumes" . 

@phoracek

Reply:
Why not? Pardon my ignorance, but you are requesting scc for a component defined under this repo, no? So you should know what type of volume it needs.

Reply:
You're right. The right thing to do - imo - is to limit this to the `hostPath` volume type - since we only mount the kubelet device plugin folder, plus the path to where we install the CNI binary.